src/backwards_compatibility.jl

```diff
@@ -35,3 +35,14 @@ end
         return isa(t, DataType) && t.mutable
     end
 end
+
+@static if :ninitialized in fieldnames(Tuple{})
```

Comment:
I don't think this needs to be `@static`, I just followed the `@static if VERSION < v"1.7.0-A"` code block above.